### DISCUSSION TITLE
Adding phpmyadmin to sail docker compose

### DISCUSSION THREAD

**Discussion by chinmaypurav** [laravel/framework#48786] [Ideas]
I need a lightweight yet powerful MySQL GUI client in my dev environment.

phpmyadmin completes this gap.

When working with Laravel Sail, I realized that it would be better if we could have a `phpmyadmin` service within the docker compose.

These would be the changes

![image](https://github.com/laravel/framework/assets/70144052/3aeca868-2f3e-4a79-bf5a-32ca72bd1052)
![image](https://github.com/laravel/framework/assets/70144052/686f35df-872b-4b4b-9a6f-751759106b5c)

I am not sure how the `curl -s https://laravel.build/example-app | bash` works, but I can push these changes to sail repo if it helps the community!

**Comment by williamdes** [comment]
Please use
```yml
image: phpmyadmin:latest
```
See: https://github.com/phpmyadmin/docker/issues/363

**Reply by chinmaypurav** [reply]
Sure.

**Comment by antimech** [comment]
[PhpMyAdmin is insecure](https://www.opencve.io/cve?vendor=phpmyadmin).

I see you using PhpStorm, then the best way is to use the really well integrated database tools, don't mind the SQLite - it supports multiple DBMS:
![](https://github.com/laravel/framework/assets/13885653/8d337e45-e78b-4a4d-8129-ca6aa9530f5f)
Configure it on [localhost:3306](https://laravel.com/docs/10.x/sail#mysql)

If you are not happy with that you could use TablePlus for Linux/MacOS/Windows, or other.

If you want to do it remotely then do it over SSH. PhpStorm does support it, you could even highlight your dev database as green, and production as red, also you could toggle read-only mode.
